apps/sports-helsinki/src/domain/search/venueSearch/VenueMapSearch.tsx

```diff
@@ -9,7 +9,7 @@ const SimpleVenueMapSearch = () => (
     searchRoute={SEARCH_ROUTES.MAPSEARCH}
     searchUtilities={null}
     korosBottom={false}
-    showTitle={false}
+    showTitle={true}
```

Comment:
```suggestion
    showTitle
```
the `{true}` is not needed and actually our linter should advice to use it without. 

_I don't know why our linters are softer nowadays (I think it's been so after the commit that had ErrorBoundary and lot's of refactoring)._